test/e2e/dra/dra.go

```diff
@@ -1910,6 +1910,22 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), func() {
 		b := drautils.NewBuilder(f, driver)
 		b.UseExtendedResourceName = true
 
+		ginkgo.It("must run a pod with implicit extended resource with one container one resource", func(ctx context.Context) {
```

Comment:
let's add a test that will confirm that a pod with both definitions works as described in https://github.com/kubernetes/kubernetes/issues/133366

Reply:
sure, added, the test has one two resources, one implicit, one explicit, both are using the same device class.